src/EnergyPlus/Furnaces.cc

```diff
@@ -688,6 +689,8 @@ namespace Furnaces {
         std::string_view constexpr getAirLoopHVACHeatCoolInput("GetAirLoopHVACHeatCoolInput");
         std::string_view constexpr routineName = "GetFurnaceInput";
 
+        using namespace OutputReportPredefined;
```

Comment:
Is this needed?  You are explicitly listing out the namespace when you are calling PreDefTableEntry, which is our current pattern, so I'm not sure this is needed.

Reply:
I agree.